src/test/fuzz/util.h

```diff
@@ -36,6 +36,17 @@
 #include <string>
 #include <vector>
 
+template <typename... Callables>
+void CallOneOf(FuzzedDataProvider& fuzzed_data_provider, Callables... callables)
+{
+    constexpr size_t call_size{sizeof...(callables)};
+    static_assert(call_size >= 1);
+    const size_t call_index{fuzzed_data_provider.ConsumeIntegralInRange<size_t>(0, call_size - 1)};
+
+    size_t i{0};
+    return ((i++ == call_index ? callables() : void()), ...);
```

Comment:
No need for an explicit `return` statement if the return type is void:

```suggestion
    ((i++ == call_index ? callables() : void()), ...);
```

I think that's clearer since you're not actually returning anything from the fold expression.

Reply:
I wanted to keep it to enforce that only void functions can be called. Though, the ternary operator already enforces that.

In the future someone might want to extend this for non-void functions, e.g. a `ConstructOneOf(...)` could create and return a type constructed according to one of several construction rules).

Will leave as-is for now. Happy to remove if I need to force-push.

Reply:
Yes, not a big deal. Can be left like this or updated if you have to touch the branch again.